src/components/TodoComponents/TodoForm.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+
+class TodoForm extends Component{
+    constructor(props) {
+        super(props);
+        this.state = {value: ''};
+    
+        this.handleChange = this.handleChange.bind(this);
+      }
```

Comment:
Using `value` as a property name works for now, but in the future, you'll want to use a more specific name (like `todoInput` or `newTodoText`, to give some ideas) especially if you're dealing with multiple input values in `state` to avoid confusion.